scripts/prepare_sector_network.py

```diff
@@ -2286,6 +2334,73 @@ def add_biomass(n, costs):
         e_initial=solid_biomass_potentials_spatial,
     )
 
+    if snakemake.params.biomass.get(
+        "include_unsustainable"
+    ) and investment_year < snakemake.params.biomass.get("unsustainable_phase_out"):
+
+        n.madd(
+            "Store",
+            spatial.gas.biogas,
+            suffix=" unsustainable",
+            bus=spatial.gas.biogas,
+            carrier="unsustainable biogas",
+            e_nom=unsustainable_biogas_potentials_spatial,
+            marginal_cost=costs.at["biogas", "fuel"],
+            e_initial=unsustainable_biogas_potentials_spatial,
+            efficiency=1,
+            e_min_pu=0,
+            e_nom_extendable=False,
+        )
+
+        n.madd(
+            "Store",
+            spatial.biomass.nodes,
+            suffix=" unsustainable",
+            bus=spatial.biomass.nodes,
+            carrier="unsustainable solid biomass",
+            e_nom=unsustainable_solid_biomass_potentials_spatial,
+            marginal_cost=14,  # mean of MINBIOWOO costs in ENS_BAU 2030
+            e_initial=unsustainable_solid_biomass_potentials_spatial,
+            efficiency=1,
+            e_min_pu=0,
+            e_nom_extendable=False,
+        )
+
+        n.madd(
+            "Bus",
+            spatial.bioliquids.nodes,
+            location=spatial.bioliquids.locations,
+            carrier="unsustainable bioliquids",
+            unit="MWh_LHV",
+        )
+
+        n.madd(
+            "Store",
+            spatial.bioliquids.nodes,
+            suffix=" unsustainable",
+            bus=spatial.bioliquids.nodes,
+            carrier="unsustainable bioliquids",
+            e_nom=unsustainable_liquid_biofuel_potentials_spatial,
+            marginal_cost=costs.at["oil", "fuel"],
+            e_initial=unsustainable_liquid_biofuel_potentials_spatial,
+            efficiency=1,
+            e_min_pu=0,
+            e_nom_extendable=False,
+        )
+
+        n.madd(
+            "Link",
+            spatial.bioliquids.nodes,
+            bus0=spatial.bioliquids.nodes,
+            bus1=spatial.oil.nodes,
+            bus2="co2 atmosphere",
+            carrier="unsustainable bioliquids",
+            efficiency=1,
+            efficiency2=-costs.at["solid biomass", "CO2 intensity"]
+            + costs.at["BtL", "CO2 stored"],
+            p_nom_extendable=True,
```

Comment:
consider setting a small capital cost or even a fixed large capacity (this can improve numerics)

Reply:
What do you think about fixing p_nom at `unsustainable_liquid_biofuel_potentials_spatial/1e3` or some other value related to the potentials?

Reply:
How about `unsustainable_liquid_biofuel_potentials_spatial` as `p_nom`. Then it's for sure unconstrained.